### PR TITLE
Sematical, Grammar, and Format Improvements For README

### PR DESCRIPTION
**Pull request by s0kil**
This PR also addresses #29.

Lots of grammar corrections, Oh mein, (:

The formatting was done with Prettier for idiomatic Markdown.

### CODE REVIEW

**Comment by snshn** [comment]
It's sema*n*tical 😅

File: README.md [hunk 1]
```diff
@@ -1436,38 +1514,41 @@ along with these options:
...
 
 along with these options:
 
 ```js
-var view = Mikado.new(template, { pool: true } );
+var view = Mikado.new(template, { pool: true });
 ```
 
-This will switch Mikado into a "cross-shared-keyed" mode.
+This will switch Mikado into an "cross-shared-keyed" mode.
```
Comment (rillig) [inline README.md]: That's wrong. The word "an" is only used if the following _sound_ is a vowel. This is clearly not the case here.

Probably a copy-and-paste mistake from "into an exclusive mode" further below.

Comment (s0kil) [reply README.md]: @ts-thomas Could you also review https://github.com/nextapps-de/mikado/issues/29#issuecomment-561680545 and I will fix both issues.

Comment (ts-thomas) [reply README.md]: Done.

Comment (s0kil) [reply README.md]: > When data is changed, the input elements will automatically update, as well as other turn around, when the input elements get new data the store will automatically update.

What does this mean "as well as other turn around" ?

Comment (s0kil) [reply README.md]: > When data is changed, the input elements will automatically update, as well as other turn around, when the input elements get new data the store will automatically update.

What does this mean "as well as other turn around" ?

Comment (ts-thomas) [reply README.md]: Sorry, just remove some words :)

> When data is changed, the input elements will automatically update, as well as ~~other turn around,~~ when the input elements get new data the store will automatically update.

**Pull request merged**